src/test/regress/expected/multi_extension.out

```diff
@@ -785,6 +787,7 @@ CREATE TABLE new_columnar_table (a int) USING columnar;
 ERROR:  loaded Citus library version differs from installed extension version
 -- do cleanup for the rest of the tests
 SET citus.enable_version_checks TO OFF;
+SET columnar.enable_version_checks TO OFF;
```

Comment:
One idea is to add an assignment hook to `citus.enable_version_checks` to also set `columnar.enable_version_checks`. The only way the settings could be different is if someone explicitly sets `columnar.enable_version_checks`. That way, these test changes would not be required.

To do this, you'd need to change the call site to `DefineCustomBoolVariable("citus.enable_version_checks", ...)` in `shared_library_init.c` to specify an assignment hook. The hook can be a short function to call `set_config_option` on `columnar.enable_version_checks`.

Reply:
We discussed this in our meeting today but want to add here in case someone from Citus team has an opinion. I'm okay with either but my concern is that down the line we would want citus.enable_version_checks to be off and columnar.enable_version_checks to be on or vice versa. Then we'd have to go back and remove the assignment hook and add call sites in our previous tests that we didn't have before. But if we don't see that being a possibility, then the assignment hook would make for cleaner code

Reply:
I'm not too concerned about making these GUCs highly usable. From a user point-of-view, it's mainly for scenarios where the check inadvertently prevents the user from doing ALTER EXTENSION, but we generally manage to avoid those.